ArrayLimitedQueue/README.md

```diff
@@ -0,0 +1,276 @@
+# ArrayLimitedQueue
+
+Array Limited Queue is a collection that has the features of a regular array (only get + limited size), queues and sets. The way you value and compare items can be user-defined.
+
+The Ordered Set data type is a hybrid of:
+- a [Queue](https://en.wikipedia.org/wiki/Queue_(abstract_data_type)), is a list where you can only insert new items at the back and remove items from the front. This ensures that the first item you enqueue is also the first item you dequeue. First come, first serve!
+- a [Set](https://en.wikipedia.org/wiki/Set_%28mathematics%29), a collection of unique items where the order does not matter, and
+- a [Sequence](https://en.wikipedia.org/wiki/Sequence), an ordered list of items where each item may appear more than once.
```

Comment:
@kelvinlauKL In the near future I will complete this work(time of diploma :)  ). Do you really think that this is a useful algorithm?